### DISCUSSION TITLE
Allow `break` keyword to break out of deeply nested loops

### DISCUSSION THREAD

**Discussion by blackears** [godotengine/godot-proposals#6580] [Scripting]
When you use the `break` keyword, it currently only breaks out of the innermost loop it is used in.  It would be useful if there was a way to be able to break out of more deeply nested loops.  Some languages use a label syntax to do this.  Maybe something like:

```
@label outer_loop
for i in my_list.size():
    while foo < 10:
        if my_condition(foo, bar):
            break outer_loop
        foo += 1
```

**Comment by AThousandShips** [comment]
Already tracked by #3780

**Comment by Calinou** [comment]
This is already being tracked in https://github.com/godotengine/godot-proposals/issues/3780.
